packages/ui-react/src/containers/ShelfList/ShelfList.tsx

```diff
@@ -38,6 +74,16 @@ const ShelfList = ({ rows }: Props) => {
     return () => setRowCount(INITIAL_ROW_COUNT);
   }, [rows]);
 
+  // Empty shelves
+  const emptyShelves = useEmptyPlaylistsReporter(rows);
+
+  if (emptyShelves) {
+    return (
+      // TODO: translate
+      <ErrorPage title="No content found" message="No videos are found. This could be a misconfiguration or the content is not available for your country." />
```

Comment:
Im leaving the spanish translation here in case you need it later:

title -  No se encontró contenido
message - No se encuentran videos. Esto podría ser una mala configuración o el contenido no está disponible para su país.